frontend/src/supplier/ProductList.css

```diff
@@ -0,0 +1,31 @@
+h2{
+    font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    margin-top: 150px;
+    padding: 0;
+}
+
+table{
+    border-collapse: collapse;
+    width: 100%;
+    margin-top: 20px;
+    font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
```

Comment:
du hast font-family in h2, table und in nem anderen css glaub im header und nochwo. Wenn du überall dieselbe Schrift benutzt kannst du auch in der app.css oder so ein Feld einfügen mit *{font-family: Helvetica blabla}